### PR TITLE
proposal: Move CPC_CHARTER to stage-2.

### PR DESCRIPTION
**Pull request by MylesBorins**
There are no longer any objections to current text.

### CODE REVIEW

**Comment by MylesBorins** [comment]
TL;DR (lmk if I missed anything)

* CPC is made up of
  - Voting members (2 members per Impact Stage project)
  - Regular members (any member of any foundation project an get added to list of regular members)
  - Observers (any member of any foundation project)
* All membership types can attend any meeting and participate in consensus process
* If there is a vote only voting members participate
* CPC has a chair and director (director attends board meetings)
* CPC has a second board seat that is allocated on an annual basis
* CPC sets foundation wide policy
* CPC is final authority to decide if projects join foundation
* CPC does not have control of technical direction of projects within the foundation
* CPC oversees budget for foundation projects
* CPC will charter a team to help with CoC escalations within foundation.

**Comment by MylesBorins** [comment]
@mhdawson's changes LGTm

File: proposals/stage-2/CPC_CHARTER/CPC-CHARTER.md [hunk 1]
```diff
@@ -74,11 +74,11 @@ can be removed by voluntary resignation, or by a standard CPC motion.
 
 Voting members are selected as follows:
 
-* Each top level project may nominate two members through a process
+* Each Impact project may nominate two members through a process
```
Comment (jorydotcom) [inline proposals/stage-2/CPC_CHARTER/CPC-CHARTER.md]: Feedback from JSF projects: We would like this to read "may nominate up to two"

Comment (mhdawson) [reply proposals/stage-2/CPC_CHARTER/CPC-CHARTER.md]: @jorydotcom added a commit to say up to two for all of the categories.

**Comment by MylesBorins** [comment]
As discussed in today's meeting I am going to land this. I followed up independently with both Jory + Michael after the meeting to ensure that we have consensus. We will move forward with this proposal and keep track of individual membership adoption. If after 6 - 12 months we notice that the program is not trending in a direction to likely trigger the seat we can revisit the allocation.
